helpers/testoff.py

```diff
@@ -0,0 +1,23 @@
+# class Klass(object):
+# 	pass
+#
+#
+#
+# setattr(Klass, 'grid_strings', ['hey'])
```

Comment:
curious - what were you trying to do here?

Reply:
:) this one suppose to be in gitignore, that is file I've tried things in, but anyway I was creating variables with dynamically created names (I do the same sometimes in ruby) for some method like: name_{arg} = value, it worked, I just found better solution